deploy/ansible/hosts/pre-devnet.yml

```diff
@@ -34,6 +34,7 @@ goshimmers:
   vars:
     analysisServerHost: "analysisentry-01.pre-devnet.shimmer.iota.cafe"
     entryNodeHost: "analysisentry-01.pre-devnet.shimmer.iota.cafe"
+    entryNodePubKey: "2PV5487xMw5rasGBXXWeqSi4hLz7r19YBt8Y1TGAsQbj"
```

Comment:
Should it have the same value as for devnet? Entry node seed is passed as parameter and pub key is a derived from that.

Reply:
Pre-devnet and Devnet currently have the same set of secrets: the aim is to for them to be "mirror" environments. Deriving pubkey from seed within Ansible would be possible to avoid this extra line, but it is painful =)